browser_use/browser/context.py

```diff
@@ -1085,13 +1085,55 @@ async def _input_text_element_node(self, element_node: DOMElementNode, text: str
 
 			# Get element properties to determine input method
 			is_contenteditable = await element_handle.get_property('isContentEditable')
+			# Check if element is a textarea
+			tag_name = await element_handle.evaluate('el => el.tagName.toLowerCase()')
+			input_type = await element_handle.evaluate('el => el.type.toLowerCase()')
+			is_textarea = tag_name == 'textarea'
+			is_radio = input_type == 'radio'
 
+			# Different handling for contenteditable, textarea, and regular input fields
+			try:
+				if await is_contenteditable.json_value():
+					await element_handle.evaluate('el => el.textContent = ""')
+					await element_handle.type(text, delay=5)
+				elif is_textarea:
+					# For textareas, handle newlines explicitly
+					await element_handle.click()
+					await element_handle.press('Control+a')  # Select all existing text
```

Comment:
This is sketchy, many apps like Google Docs or Hedgedoc hijack keyboard shortcuts and will delete everything on the page if you do this.

Better to clear the element contents with JS or playwright than try to use keyboard shortcuts.
(also you'd want `ControlOrMeta+a` so it works on mac/linux/windows)